constants/faqs.ts

```diff
@@ -24,38 +24,60 @@ export const CXC_FAQ: QandA[] = [
 export const GENERAL_FAQ: QandA[] = [
   {
     id: "1",
-    question: "Lorem ipsum dolor sit amet, consectetur adipiscing elit?",
-    answer:
-      "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco.",
+    question: "What is DSC?",
+    answer: `We are a MathSoc club dedicated to building a community of students 
+passionate about exploring the field of Data Science. DSC hosts a number
+of workshops/events throughout the term, mostly academic focused (see below). 
+Our iconic mascots include Echo the Whale and Bert (ref the ML model by Google)`,
   },
   {
     id: "2",
-    question: "Lorem ipsum dolor sit amet, consectetur adipiscing elit?",
-    answer:
-      "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco.",
+    question: "What are some events DSC hosts?",
+    answer: `1. Intro workshops on data tools which will look great on your resume :) 
+E.g. Pandas, Tableau, Build Your Own DS Project, Machine Learning Algorithms) \n
+2. Industry professionals/alumni/upper-year panels to help students explore more of 
```

Comment:
using \n is alright for now, but I think it causes a bit too much spacing? in the future we could make each answer an array of strings, and render an array of paragraphs so we can customize spacing or smth like that